### PR TITLE
RELATED: BB-2296 empty preview when missing format

### PR DESCRIPTION
**Pull request by strouhaljan**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7393/ : SUCCESS in 3m 14s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3416/ : SUCCESS in 3s

File: src/components/measureNumberFormat/customFormatDialog/shared/tests/FormattedPreview.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        const wrapper = renderComponent({ format: "[>1][green]#,##" });
+        expect(wrapper.props().style).toMatchObject({ color: "#00AA00" });
+    });
+
+    it("should render formatted number without colors when coloring is disabled", () => {
+        const wrapper = renderComponent({ format: "[>1][GREEN]#,##", colors: false });
+        expect(wrapper.props().style).toEqual(undefined);
+    });
+
+    it("should null value if no value is provided", () => {
```
Comment (xMort) [inline src/components/measureNumberFormat/customFormatDialog/shared/tests/FormattedPreview.spec.tsx:40]: typo?

**Comment by strouhaljan** [comment]
^^ in the last force-push, I've also implemented some of the changes requested by Petr Kaspar.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7403/ : FAILURE in 3m 54s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3428/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7406/ : SUCCESS in 3m 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3431/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7410/ : SUCCESS in 3m 18s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3433/ : SUCCESS in 3s

**Comment by xMort** [comment]
LGTM

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3867/ : UNSTABLE in 12m 17s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-fossa-scanning-zuul-el7/26/ : SUCCESS in 18s (non-voting)
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7411/ : SUCCESS in 7m 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1801/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2647/ : SUCCESS in 17m 29s

**Comment by xMort** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3869/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3436/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3226/ : UNSTABLE in 11m 35s

**Comment by gdgate** [comment]
Build in pipeline gate aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7416/ : SUCCESS in 7m 03s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3437/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3874/ : SUCCESS in 12m 08s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-fossa-scanning-zuul-el7/32/ : SUCCESS in 19s (non-voting)
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7419/ : SUCCESS in 7m 19s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1807/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2654/ : SUCCESS in 18m 21s

**Pull request merged**